guides/common/modules/proc_configuring-alternate-content-sources.adoc

```diff
@@ -0,0 +1,54 @@
+[id="Configuring_Alternate_Content_Sources_{context}"]
+= Configuring Alternate Content Sources
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Alternate Content Sources* > *Add Source*.
+. Set the *Source type* as *Custom*.
+. Select the *Content type*.
+. Enter a name in the *Name* field and optionally provide a description in the *Description* field.
+. Select {SmartProxies} to which the alternate content source is to be synced.
+. Enter the Base URL of the alternate content source.
+. Enter a comma-separated list of Subpaths.
+. Provide the *Manual Authentication* or *Content Authentication* credentials, if these are needed.
+. If SSL verification is required, enable *Verify SSl*, and select the SSL CA certificate.
+. Review details and click *Add*.
+. Refresh the alternate content sources by executing the following command:
```

Comment:
I did not test setting up an ACS yet, but does this mean it always requires an interaction with Foreman Server through CLI?

Reply:
Hey @ianballou can you weigh in on this? Do we need to refresh the ACS through CLI every time?

Reply:
ACSs need to be refreshed before they are used. That being said, there are multiple ways to refresh an ACS:

1) Refresh via the UI
2) Refresh via hammer
3) Refresh via the ACS refresh Cron job that runs weekly by default

Reply:
Hey @ianballou so to refresh via UI, will the user only have to refresh the browser page, right?

Reply:
@maximiliankolb I am checking this further with Ian and will push a new commit including the UI procedure for refreshing ACS.